.changeset/smooth-chicken-wash.md

```diff
@@ -2,4 +2,89 @@
 'astro': minor
 ---
 
-Implements Content Layer
+Adds experimental support for the content layer API.
+
+The Content Layer API is a new way to handle content in Astro. It builds upon the content collections, taking them beyond local files in `src/content` and allowing you to fetch content from anywhere, including remote APIs, or files anywhere in your project. As well as being more powerful, the Content Layer is designed to be more performant, helping sites scale to thousands of pages. Data is cached between builds and updated incrementally. Markdown parsing is also 5-10x faster, and uses much less memory.
+
+You can try content layer with your existing content, or try a custom loader.
```

Comment:
```suggestion
You can try content layer with your existing content, or try a custom loader.

To enable the content layer, add the following experimental flag in your Astro config:

```js
{
  experimental: {
      contentLayer: true,
   }
}
```

I'm not sure which flag name you're using, but the changeset should show people how to add it to their config!

Reply:
There's no flag. It's enabled by using `type: "experimental_content"` rather than a config flag

Reply:
Ah, I see this now. OK, I'll think about this then!